react/features/welcome/components/WelcomePageLists.js

```diff
@@ -1,139 +0,0 @@
-// @flow
-
-import React, { Component } from 'react';
-
-import { translate } from '../../base/i18n';
-import { IconEventNote, IconRestore } from '../../base/icons';
-import { PagedList } from '../../base/react';
-import { connect } from '../../base/redux';
-import { CalendarList, isCalendarEnabled } from '../../calendar-sync';
-import { RecentList } from '../../recent-list';
-import { setWelcomePageListsDefaultPage } from '../actions';
-
-/**
- * The type of the React {@code Component} props of {@link WelcomePageLists}.
- */
-type Props = {
-
-    /**
-     * Whether the calendar functionality is enabled or not.
-     */
-    _calendarEnabled: boolean,
```

Comment:
Does the new implementation consider this?

Reply:
Forget about this. Right now it does. Thanks!